app-tasks/rf/src/rf/commands/export.py

```diff
@@ -93,82 +83,6 @@ def get_export_definition(export_uri):
-    copied = deepcopy(export_definition)
-    # Only two slashes in file:// because local_dir is an absolute unix path
-    # copied['output']['source'] = 's3://rasterfoundry-development-data-us-east-1/chris-test-local/'
-    copied['output']['source'] = 'file://{}'.format(local_dir)
-    # copied['output']['source'] = 'file:/{}.tif'.format(local_dir)
-    with open(local_ed_path, 'w') as outf:
-        outf.write(json.dumps(copied))
-    return local_ed_path
-
-
-def post_process_exports(export_definition, tiff_directory):
-    """Run GDAL commands to make exports sane
-
-    Returns: path to final tif file
-    """
-    jwt = get_jwt()
-    root_url = os.getenv('RF_HOST')
-    parsed_url = urlparse(root_url)
-    api = API(api_token=jwt, host=parsed_url.netloc, scheme=parsed_url.scheme)
-    logger.info('Retrieving Export: %s', export_definition['id'])
-    export_obj = api.client.Imagery.get_exports_exportID(exportID=export_definition['id']).result()
-    temp_geojson = os.path.join(tiff_directory, 'cut.json')
-    with open(temp_geojson, 'wb') as fh:
-        geojson = {
-            'type': 'FeatureCollection',
-            'features': [
-                {
-                    "type": "Feature",
-                    "properties": {},
-                    "geometry": export_obj.exportOptions.mask
-                }
-            ]
-        }
-        json.dump(geojson, fh)
-
-    tiff_files = [os.path.join(tiff_directory, f) for f in os.listdir(tiff_directory) if f.endswith('tiff')]
-    logger.info('Found %s files to merge', len(tiff_files))
-    merged_tiff = os.path.join(tiff_directory, 'combined.tiff')
-    translated_tiff = os.path.join(tiff_directory, 'translated.tiff')
-    export_tiff = os.path.join(tiff_directory, 'export.tiff')
-    merge_command = ['gdal_merge.py', '-o', merged_tiff] + tiff_files
-    warp_command = []
-    if export_obj.exportOptions.mask is None:
-        warp_command = ['gdalwarp', '-co', 'COMPRESS=DEFLATE',
-                        '-co', 'TILED=YES', merged_tiff, export_tiff]
-    else:
-        warp_command = ['gdalwarp', '-co', 'COMPRESS=DEFLATE',
-                        '-co', 'TILED=YES', '-crop_to_cutline',
-                        '-cutline', temp_geojson, merged_tiff, export_tiff]
-    translate_command = ['gdal_translate', '-co', 'COMPRESS=DEFLATE', '-co', 'TILED=YES', '-stats',
-                         export_tiff, translated_tiff]
-    logger.info('Running Merge Command: %s', ' '.join(merge_command))
-    subprocess.check_call(merge_command)
-    logger.info('Running Warp Command: %s', ' '.join(warp_command))
-    subprocess.check_call(warp_command)
-    logger.info('Running Translate Command: %s', ' '.join(translate_command))
-    subprocess.check_call(translate_command)
-    logger.info('Finished post-processing export: %s', export_tiff)
-    return translated_tiff
-
-
-def upload_processed_tif(merged_tiff_path, export_definition):
```

Comment:
I don't think we can get rid of this unless we're adding dropbox export back to the scala export process